core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1156,6 +1158,433 @@ describe('Contributions and review component', () => {
         });
       });
 
+      it('should load reviews, then contributions', fakeAsync(() => {
```

Comment:
```suggestion
      it('should show only selected contribution type when switching tabs', fakeAsync(() => {
```

Reply:
Changed to "should show only selected type when switching tabs." due to length.